test/test_datasets.py

```diff
@@ -28,7 +30,7 @@
     HAS_PYAV = False
 
 
-class Tester(unittest.TestCase):
+class DatasetTestcase(unittest.TestCase):
```

Comment:
This was necessary since `STL10Tester` needs access to `generic_classification_dataset_test` without subclassing `Tester` directly. Otherwise running `STL10Tester` would also run all inherited test.